test/include/interfacekernels/InterfaceReaction.h

```diff
@@ -0,0 +1,28 @@
+
+#ifndef INTERFACEREACTION_H
```

Comment:
Remove this and replace it with `#pragma once`. This was part of a recent change to how we construct our header files.